packages/lesswrong/lib/rateLimits/constants.ts

```diff
@@ -27,13 +28,13 @@ const EA: {POSTS: PostAutoRateLimit[], COMMENTS: CommentAutoRateLimit[]} = {
   COMMENTS: [
     {
       ...timeframe('4 Comments per 30 minutes'),
-      karmaThreshold: 30,
+      isActive: user => user.karma <= 30,
```

Comment:
having `=> user.karma <=` without any parentheses is confusing when skimming; I'd parenthesize the `user` param at a minimum.

(also, for consistency's sake, please stick to one of `<=` or `<` for karma thresholds; having both spread across different rate limits is very likely to lead to annoying off-by-one errors)